py3status/i3status.py

```diff
@@ -241,7 +215,7 @@ def set_time_zone(self, item):
         ) - datetime(utcnow.year, utcnow.month, utcnow.day, utcnow.hour, utcnow.minute)
         # create our custom timezone
         try:
-            self.tz = Tz(i3s_time_tz, delta)
+            self.tz = timezone(delta, i3s_time_tz)
         except ValueError:
```

Comment:
It looks like you wouldn't need try/except `ValueError` too.

Reply:
Both versions seem to raise it if the offset exceeds some value.